support/power-platform/power-apps/ribbon-issues-button-hidden.md

```diff
@@ -41,6 +41,9 @@ Once the Command Checker tool has been enabled, within the application in each o
 
     :::image type="content" source="media/ribbon-issues-button-hidden/new-hidden.png" alt-text="Screenshot shows the New button on the contact entity's grid page is not visible and is represented by an item labeled New (hidden).":::
 
+    > [!NOTE]
+    > If you see all rules evaluate to true, but the button is still hidden, it may be due to [context sensitive commands in grids](https://support.microsoft.com/en-us/topic/button-in-command-bar-does-not-appear-after-grid-item-selection-1100b6b5-ab64-6047-82c5-924a2b38296a). When records are selected on a grid, all buttons without a SelectionCountRule will be considered not relevant to the selected record(s) and be hidden even if their rule evaluation is true.  Note this does not effect flyouts, since flyout children might still have record based commands.
```

Comment:
I think the URL should not contain the language locale "en-us/" part to allow the locale to automatically resolved

Reply:
Removed, good point.